### PR TITLE
Refactor TopicMessages's store to redux/toolkit

### PR DESCRIPTION
**Pull request by rAzizbekyan**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [x] Unit checks
- [ ] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by workshur — APPROVED** [review]

**Comment by workshur** [comment]
@rAzizbekyan looks good to me. Please ask @Haarolean to test messages UI before merge

**Review by simonyandev — CHANGES_REQUESTED** [review]
I can see a lot of thunk actions are left and only two of them are transformed into createAsyncThunk. All those thunks need to become createAsyncThunk and all those action types like 'GET_TOPIC_CONSUMER_GROUPS__FAILURE' must be removed, the toolkit will handle them. Also, make sure they are not left in other places too like where loaders and alerts are handled.

**Review by simonyandev — APPROVED** [review]
Didn't know that there's a separate task for Topics reducer. The rest is good for Topic Messages reducer👍

**Comment by Haarolean** [comment]
> @rAzizbekyan looks good so far. Please ask @Haarolean to test messages UI before merge

ok we'll schedule a QA round for this

File: kafka-ui-react-app/src/redux/reducers/topicMessages/topicMessagesSlice.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    },
+    resetTopicMessages: () => initialState,
+    updateTopicMessagesPhase: (state, action) => {
+      state.phase = action.payload;
+    },
+    updateTopicMessagesMeta: (state, action) => {
+      state.meta = action.payload;
+    },
+    setTopicMessagesFetchingStatus: (state, action) => {
+      state.isFetching = action.payload;
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/redux/reducers/topicMessages/topicMessagesSlice.ts:74]: a question didn't we have a loader slicer that did this for us , where we added fetch and other state ? so we did not have to keep a loader state in a specific slice logic

Comment (rAzizbekyan) [reply kafka-ui-react-app/src/redux/reducers/topicMessages/topicMessagesSlice.ts:74]: I got you but the problem here is that we don't have separate action for fetching topicMessages as you can see that's why we have to keep this logic for this case in slice

**Review by lazzy-panda — APPROVED** [review]

**Pull request merged**